packages/contracts/contracts/libraries/bridge/CrossDomainEnabled.sol

```diff
@@ -9,7 +9,6 @@ import { ICrossDomainMessenger } from "./ICrossDomainMessenger.sol";
  * @dev Helper contract for contracts performing cross-domain communications
  *
  * Compiler used: defined by inheriting contract
```

Comment:
Could you also remove this `Compiler used` string in this contract and in any other contracts where this appears?